.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: scala
 jdk:
 - oraclejdk8
-script: sbt +test
+script: 
+  - sbt +test
```

Comment:
Because otherwise we cannot use the following in the future:

```yaml
   - 2.9.3
   - 2.10.4
   - 2.11.2
```

This is the reason I tend to restore the original Travis command when I add additional build steps.